external/fv3fit/fv3fit/reservoir/config.py

```diff
@@ -1,16 +1,16 @@
 import dacite
-from dataclasses import asdict, dataclass
+from dataclasses import dataclass, asdict
+from typing import Tuple, Sequence
 import fsspec
 from typing import Optional
 import yaml
 
 
 @dataclass
-class SubdomainConfig:
```

Comment:
This is the only new code, I left this change in this PR so that the `ReservoirTrainingConfig` used for cubedsphere data can stay in this PR.